qutebrowser/misc/editor.py

```diff
@@ -96,11 +96,12 @@ def on_proc_closed(self, exitcode, exitstatus):
     def on_proc_error(self, _err):
         self._cleanup()
 
-    def edit(self, text):
+    def edit(self, text, caret_position):
```

Comment:
This breaks other places calling `edit`, such as the `:edit-url` command (see [failing tests](https://travis-ci.org/qutebrowser/qutebrowser/jobs/287704068)). It should probably just default to 1.